e2e/runner/emissions.go

```diff
@@ -0,0 +1,90 @@
+package runner
+
+import (
+	"fmt"
+
+	"cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	"github.com/zeta-chain/node/cmd/zetacored/config"
+	"github.com/zeta-chain/node/e2e/txserver"
+	e2eutils "github.com/zeta-chain/node/e2e/utils"
+	emissionstypes "github.com/zeta-chain/node/x/emissions/types"
+	observertypes "github.com/zeta-chain/node/x/observer/types"
+)
+
+// FundEmissionsPool funds the emissions pool on ZetaChain with the same value as used originally on mainnet (20M ZETA)
+func (r *E2ERunner) FundEmissionsPool() error {
+	r.Logger.Print("⚙️ funding the emissions pool on ZetaChain with 20M ZETA (%s)", txserver.EmissionsPoolAddress)
```

Comment:
Do we need log + 1 one function call as a separate function? Consider removing this

Reply:
I think it's fine to keep it. The logs for the test aren't too much right now